src/soil_funcs.cxx

```diff
@@ -63,28 +65,54 @@ extern double calc_Geff(bool use_closed_form_G, double theta1, double theta2, do
       Se = calc_Se_from_h(h_f,vg_alpha,vg_m,vg_n);
       printf("Se_f = %8.6lf,  Se_inverse = %8.6lf\n", Se_f, Se);
     }
-    
+
     // nint = number of "dh" intervals to integrate over using trapezoidal rule
-    dh = (h_f-h_i)/(double)nint;
+    dh = -1*(h_f-h_i)/(double)nint;
+    dh = dh*0.01;
```

Comment:
`dh` is always positive, please explain `-1` and `0.01`

Reply:
The 0.01 is just a factor making dh very small to begin with; in the event that the integration begins at a sensitive region of the soil water retention curve (which it often will, as the limits of integration go from low magnitude psi to high magnitude psi), we want the initial dh to be quite small and it expands if it can. So that's just for initialization. As for the negative, you're right that dh should be positive. I wrote -1*(h_f-h_i) but meant (h_i-h_f). Because h_i>h_f, these are equivalent. Rewriting to (h_i-h_f) for clarity.